### PR TITLE
[WB-5300] Merge/0.10.29

### PR DESCRIPTION
**Pull request by raubitsj**
https://wandb.atlassian.net/browse/WB-5300

Description
-----------

Merge changes for WB-5300 from 0.10.29 release branch into master

Testing
-------

unittests

### CODE REVIEW

File: wandb/sdk/internal/file_stream.py [hunk 1]
```diff
@@ -236,6 +241,7 @@ def _thread_body(self):
         posted_anything_time = time.time()
         ready_chunks = []
         finished = None
+        last_log = 0
```
Comment (KyleGoyette) [inline wandb/sdk/internal/file_stream.py:244]: I think this is a remnant of a change I was making that I discarded

**Review by KyleGoyette — COMMENTED** [review]
There's an unused variable `last_log`, that I believe I added and forgot to remove. But otherwise this looks good.

File: wandb/sdk/internal/file_stream.py (hunk 1 above)
Comment (raubitsj) [reply wandb/sdk/internal/file_stream.py]: @KyleGoyette , removed, thanks

**Review by KyleGoyette — APPROVED** [review]
Looks good. Lets hope this sheds some light

**Pull request merged**